experimenter/experimenter/experiments/models.py

```diff
@@ -1329,6 +1329,47 @@ def audience_overlap_warnings(self):
 
         return warnings
 
+    def get_invalid_fields_errors(self, data=None):
+        from experimenter.experiments.api.v5.serializers import (
+            NimbusReviewSerializer,
+        )
+
+        temp_data = NimbusReviewSerializer(self).data
```

Comment:
`temp_data` isn't a very descriptive variable name.  I might've called this `experiment_data` or `serializer_data` or something.  In general think about what the contents of the variable are, where it comes from, how it's used, etc to think of names.